expression/aggregation/base_func.go

```diff
@@ -233,6 +235,12 @@ func (a *baseFuncDesc) typeInfer4LeadLag(ctx sessionctx.Context) {
 	}
 }
 
+func (a *baseFuncDesc) typeInfer4VarPop(ctx sessionctx.Context) {
+	//var_pop's return value type is double
```

Comment:
```suggestion
	// var_pop's return value type is double
```

It's recommended to add a space between `//` and the first word.